.github/workflows/send-dso-metrics.yml

```diff
@@ -10,6 +10,7 @@ permissions:
 jobs:
   SendDSOEvents:
     runs-on: ubuntu-latest
+    environment: prod
```

Comment:
Just peeking at this, can't comment directly but I think you've got a typo on line 33: mv-review -> mc-review

Reply:
oh lol yeah, good eyes -- thanks!